### PR TITLE
autotest: move all default params to default_params directory

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

**Comment by khancyr** [comment]
The point is to clear a little autotest directory.  There are plenty stuff here...

**Comment by lucasdemarchi** [comment]
@rmackay9 @tridge @peterbarker any opinion?

**Comment by magicrub** [comment]
no objection from me

**Comment by peterbarker** [comment]
@lucasdemarchi I'm probably the only person who has referenced the current locations outside the ArduPilot source tree, and that only in documentation for dronekit-python.

I have no objections.

 It would be nice to do some renaming while we are here.  In particular, copter_params.parm seems a little redundant.

**Comment by khancyr** [comment]
ok for renaming ! 
By the way, why use .parm instead of .param ?

File: Tools/autotest/arducopter.py [hunk 1]
```diff
@@ -928,7 +928,7 @@ def fly_ArduCopter(binary, viewerip=None, map=False, valgrind=False, gdb=False):
     mavproxy.expect('Received [0-9]+ parameters')
 
     # setup test parameters
-    mavproxy.send("param load %s/copter_params.parm\n" % testdir)
+    mavproxy.send("param load %s/default_params/copter_params.parm\n" % testdir)
```
Comment (lucasdemarchi) [inline Tools/autotest/arducopter.py]: @rmackay9 are you ok with this being copter.parm as suggested above?

Comment (rmackay9) [inline Tools/autotest/arducopter.py]: @lucasdemarchi, yes, sounds fine to me thanks!

**Comment by lucasdemarchi** [comment]
I fixed the conflicts and applied. The file renames will be in a separate PR since there were some substantial changes and not only the suggested copter_params.param->copter.parm.

**Pull request closed without merging**